lib/engine/game/g_1877_stockholm_tramways/entities.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G1877StockholmTramways
+      module Entities
+        COMPANIES = [].freeze
+
+        CORPORATIONS = [
+          {
+            float_percent: 50,
+            sym: 'SS',
+            name: 'AB Stockholms Spårvägar',
+            logo: '1877_stockholm_tramways/SS',
+            shares: [20, 10, 10, 10, 10, 10, 10, 10, 10],
+            tokens: [0, 60, 60, 60, 60, 60],
+            coordinates: 'F6',
+            city: 0,
+            color: '#9A9A9D',
+            text_color: :black,
+            reservation_color: nil,
```

Comment:
These nil lines can probably be left out, to shorten files a bit, but it probably does not matter at all.

Reply:
Just kept them because many other games did.